parachain/primitives/beacon/src/lib.rs

```diff
@@ -385,7 +408,7 @@ pub struct SyncAggregate<SyncCommitteeSize: Get<u32>, SignatureSize: Get<u32>> {
 )]
 #[scale_info(skip_type_params(FeeRecipientSize, LogsBloomSize, ExtraDataSize))]
 #[codec(mel_bound())]
-pub struct ExecutionPayload<
+pub struct ExecutionPayloadHeaderCapella<
```

Comment:
The name `ExecutionPayloadHeaderCapella` is a bit verbose. The `Capella` isn't necessary, for the reasons we spoke of offline.

```suggestion
pub struct ExecutionPayloadHeader<
```